.changes/v3.8.0/899-features.md

```diff
@@ -0,0 +1,2 @@
+* **New Resource:** `vcd_catalog_vapp_template` to manage the upload and usage of vApp Templates [GH-899]
```

Comment:
This comment is about PR description (putting it here for easier flow):

- Please add the explanation of "the why" (why we are doing this) as one of the first sentences of the PR. This is the first question I get if I read the description with "fresh user eyes".

- >  (see Quirks section below)

There is no "Quirks" section :)

Thanks!

Reply:
Yeah, you're right. I kept updating and changing the description and forgot to polish some details. I've fixed it, let me know now if it's more understandable now.